optuna/storages/_rdb/models.py

```diff
@@ -522,6 +572,58 @@ def all(cls, session: orm.Session) -> List["TrialValueModel"]:
         return session.query(cls).all()
 
 
+class TrialIntermediateValueModel(BaseModel):
```

Comment:
Do you think if it is possible to add unit tests for `TrialIntermediateValueModel`?